config/nightwatch.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
   test_workers: false,
   test_settings: {
     'default': {
+      globals: {
+        buildtype: process.env.BUILDTYPE
```

Comment:
This makes sense; 👍  if it's clear and works for everyone.

There's an existing pattern (https://github.com/department-of-veterans-affairs/vets-website/blob/master/test/messaging/01-required-desktop.e2e.spec.js#L7) that checks the env's buildtype before it registers the tests with nightwatch. That version targets the dev environment explicitly, but the general pattern is similar. 

Given that the e2e's are run against a build, it might be nice to have a single place (maybe a `config/settings.js`) to set that global to apply it to both and allow overrides:

```
// Set defaults based on buildtype

defaults = {
    development: {
       feature_1990e: true,
       ...
    },
    ...
}[process.env.BUILDTYPE]

// Override defaults if specified by environment
export {
   ...defaults,
   ...removeEmpty({
     feature_1990e: process.env.FEATURE_1990E,
     ...
   })
}
```

though it may be unnecessarily complex. We could also have the build expose information about what features it has enabled so the e2e's can determine dynamically which features to test based on the build. We've had a case or two where we forgot to remove the guards from the e2e's when the feature was launched this approach might help with.

cc @robbiethegeek-usds